### DISCUSSION TITLE
Upgrading to 3.1.11

### DISCUSSION THREAD

**Discussion by maxzapp** [netbox-community/netbox#9125] [Q&A]
Hi,

I am on 3.1.8 and i want to upgrade to 3.1.11.
According the the docs i have to 

sudo git checkout v3.1.11

But i got an error:pathspec v3.1.11 did not match any files known to git.

Additionally, if it would work how would the second command look

sudo git pull origin master
Or
sudo git pull origin v3.1.11

Thx to all

**Comment by dreng** [comment]
I'm not sure what's causing this behaviour but v3.1.11 is not a tag according to my local git clone. Use "git checkout v3." and press the tab key twice to see all valid tags. Alternatively you can use "git log --tags --no-walk".

Strange thing is that the tag shows up here: https://github.com/netbox-community/netbox/tags

**Reply by candlerb** [reply]
Works for me:

```
$ git tag | grep 'v3\.1'
v3.1-beta1
v3.1.0
v3.1.1
v3.1.10
v3.1.11
v3.1.2
v3.1.3
v3.1.4
v3.1.5
v3.1.6
v3.1.7
v3.1.8
v3.1.9
```

Try "git fetch".  Or perhaps it's because you did your initial checkout with `--depth 1`? If so, try making a fresh `git clone` without this.

**Comment by fonoro** [comment]
I am having the same issue. Trying to upgrade from v3.3.2 to v3.3.10. However, when I issue a `/opt/netbox$ sudo git tag`, I get tags ending on v3.1.7

**Reply by candlerb** [reply]
Type "sudo git branch" to see what branch you are on.  If it's not master then do "sudo git checkout master". Then do "sudo git pull"

**Reply by fonoro** [reply]
Thank you ! That did it for me.
